File: wallet/src/main/java/com/worldventures/wallet/analytics/settings/AboutAnalyticsAction.java
package com.worldventures.wallet.analytics.settings;

import com.worldventures.janet.analytics.AnalyticsEvent;
import com.worldventures.core.service.analytics.AdobeTracker;
import com.worldventures.wallet.analytics.WalletAnalyticsAction;

@AnalyticsEvent(action = "wallet:settings:general:about",
                navigationState = true,
                trackers = AdobeTracker.TRACKER_KEY)
public class AboutAnalyticsAction extends WalletAnalyticsAction {
}

File: wallet/src/main/java/com/worldventures/wallet/analytics/settings/AboutAnalyticsAction.kt
package com.worldventures.wallet.analytics.settings

import com.worldventures.janet.analytics.AnalyticsEvent
import com.worldventures.core.service.analytics.AdobeTracker
import com.worldventures.wallet.analytics.WalletAnalyticsAction

@AnalyticsEvent(action = "wallet:settings:general:about", navigationState = true, trackers = arrayOf(AdobeTracker.TRACKER_KEY))
class AboutAnalyticsAction : WalletAnalyticsAction()
